openapi/paths/aml-checks@{id}@start-review.yaml

```
@@ -0,0 +1,33 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+post:
+  x-products:
+    - Core
+  tags:
+    - AML
+  summary: Start review of an AML check
+  operationId: PostAmlCheckStartReview
+  x-sdk-operation-name: startReview
+  description: |-
+    Starts the review process of an AML check with a specified ID.
```

suggestion: I'm not sure if this is for manual review (suggestion is based on starting a manual review)
```suggestion
    Starts the manual review process for an AML check with a specified ID.
```